### ISSUE TITLE
Port to Hercules for standalone

### ISSUE THREAD

**Issue by malloryprow** [NOAA-EMC/EMC_verif-global#122]
EMC_verif-global needs to be able to run standalone on Hercules

**Comment by malloryprow** [comment]
@DavidHuber-NOAA Was this ever done or is it a part of the ongoing work of https://github.com/NOAA-EMC/EMC_verif-global/issues/198

**Comment by DavidHuber-NOAA** [comment]
Yes, I tested #198 on Hercules as well. I'll update the description so it closes this issue when merged.

**Comment by DavidHuber-NOAA** [comment]
Closing in favor of #198.
